mava/configs/env/matrax.yaml

```diff
@@ -0,0 +1,26 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+
+env_name: Matrax
+
+scenario:
+  task_name: Climbing-stateless-v0
+
+eval_metric: episode_return
+
+# Possible scenarios:
```

Comment:
NIT: Please move this up to be above `scenario` to be consistent with the other env configs.